src/data/full-quiz.js

```diff
@@ -1,4 +1,13 @@
 const fullQuiz = [
+  {
+    Question: "Select the right syntax of the Arrow function (ES6)?",

```

Comment:
I think we should replace the `?` mark with a colon since this is not a question.

Proposed fix: 
Select the right syntax for an ES6 arrow function: